js/src/components/google-combo-account-card/getAccountCreationTexts.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	CREATING_ADS_ACCOUNT,
+	CREATING_BOTH_ACCOUNTS,
+	CREATING_MC_ACCOUNT,
+} from './constants';
+
+/**
+ * Account creation in progress description.
+ * @param {string|null} creatingWhich Which account is being created.
+ * @return {Object} Text and subtext.
+ */
+const getAccountCreationTexts = ( creatingWhich ) => {
+	let text = null;
+	let subText = null;
+
+	switch ( creatingWhich ) {
+		case CREATING_BOTH_ACCOUNTS:
+			text = __(
+				'You don’t have Merchant Center nor Google Ads accounts, so we’re creating them for you.',
+				'google-listings-and-ads'
+			);
+			subText = __(
+				'Merchant Center is required to sync products so they show on Google. Google Ads is required to set up conversion measurement for your store.',
+				'google-listings-and-ads'
+			);
```

Comment:
Agreed. This is a good spot, but I suspect that this was a design oversight and it would be best to keep consistency between all of the AccountCard designs. If that needs to change we can handle it in a follow-up issue.

Reply:
Got confirmation that this was a design inconsistency, which is being updated in the Figma file. 👍🏻